### DISCUSSION TITLE
Welcome to gorm-tools Discussions!

### DISCUSSION THREAD

**Discussion by basejump** [yakworks/gorm-tools#277] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jromero-onclick** [comment]
Hello! I have just discovered this awesome grails plugin!
What is your current future plan?
Are you interested in to be compatible with Grails 7?
